### ISSUE TITLE
Windows CI build error related to conda

### ISSUE THREAD

**Issue by matthewmturner** [ibis-project/ibis#2493]
The Windows error seems to be a new genuine error. Unrelated to this PR, but we'll have to fix it before merging. If you want to open an issue, that would be great.

The Impala error seems to be the error we've experienced several times, I guess it'll go away after a rerun or two.

_Originally posted by @datapythonista in https://github.com/ibis-project/ibis/pull/2491#issuecomment-715308115_

**Comment by matthewmturner** [comment]
Errors from azp.

1.
```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - conda=4.6 -> python[version='>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']

Your python: python=3.8

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

2.
```
Starting: Publish Windows environment YAML to Azure
==============================================================================
Task         : Publish Pipeline Artifacts
Description  : Publish (upload) a file or directory as a named artifact for the current run
Version      : 1.2.3
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact
==============================================================================
Artifact name input: WindowsCondaEnvironment37
##[error]Path does not exist: D:\a\1\s\env.yml
Finishing: Publish Windows environment YAML to Azure
```

**Comment by datapythonista** [comment]
Thanks @matthewmturner, I think this was fixed in #2486. Can you rebase the PR where you found this, and let me know if it still happens (and you can close if it doesn't).
